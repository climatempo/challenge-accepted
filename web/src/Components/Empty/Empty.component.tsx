import { Typography } from '@mui/material';

function Empty() {
    return <div className="flex flex-col justify-center items-center pt-8 pb-16 sm:py-20 text-center gap-8">
        <Typography variant="h2" className="hidden md:block">Clima Tempo Challenge</Typography>
        <Typography variant="h6" className="hidden md:block">Use a barra de pesquisa para achar sua cidade.</Typography>
        <Typography variant="h4" className="md:hidden">Clima Tempo<br/>Challenge</Typography>
        <Typography variant="h6" className="md:hidden">Use a barra de pesquisa para achar sua cidade.</Typography>
    </div>;
}

export default Empty;