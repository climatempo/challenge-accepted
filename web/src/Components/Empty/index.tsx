import Empty from './Empty.component';

export default Empty;