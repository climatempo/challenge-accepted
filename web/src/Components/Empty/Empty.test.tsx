import renderer from 'react-test-renderer';
import Empty from './index';


test('Empty component', () => {
    const empty = renderer.create(<Empty/>);

    expect(empty.toJSON()).toMatchSnapshot();
});