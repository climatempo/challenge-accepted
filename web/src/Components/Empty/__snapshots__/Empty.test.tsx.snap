// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty component 1`] = `
<div
  className="flex flex-col justify-center items-center pt-8 pb-16 sm:py-20 text-center gap-8"
>
  <h2
    className="MuiTypography-root MuiTypography-h2 hidden md:block css-1sra7t5-MuiTypography-root"
  >
    Clima Tempo Challenge
  </h2>
  <h6
    className="MuiTypography-root MuiTypography-h6 hidden md:block css-2ulfj5-MuiTypography-root"
  >
    Use a barra de pesquisa para achar sua cidade.
  </h6>
  <h4
    className="MuiTypography-root MuiTypography-h4 md:hidden css-5lbw0b-MuiTypography-root"
  >
    Clima Tempo
    <br />
    Challenge
  </h4>
  <h6
    className="MuiTypography-root MuiTypography-h6 md:hidden css-2ulfj5-MuiTypography-root"
  >
    Use a barra de pesquisa para achar sua cidade.
  </h6>
</div>
`;
